console.rb

```diff
@@ -4,14 +4,93 @@
 
 # NOTE: Uncomment the lines below to verify that your seed script is working
 
+#
+#
 # puts "There are #{Apartment.count} apartments"
 # puts "There are #{Tenant.count} tenants"
 #
 # puts "*" * 50
 #
 # puts "The first apartment is at #{Apartment.first.address}."
 # puts "It has  #{Apartment.first.tenants.count} tenants."
+#
+# binding.pry
+#
+# puts "ignore this line, it's just here so the binding.pry above works"
+
+puts "
+          Welcome to Tenant / Apartment Management System
+          Input following commands to proceed
+          "
+      userInput=nil
+
+
+      userInput = 'menu'
+              while !(userInput==nil)
+                case userInput
+                when "1"
+                   Apartment.all.each {|ap| puts "Property:# #{ap.id}; Located at: #{ap.address}"}
+                   userInput = gets.chomp
+                when "2"
+                   Tenant.all.each {|t| puts "Tenant ID: #{t.id}; Tenant Name: #{t.name};"}
+                   puts "You are back to the top menu, 'menu' for options, 'exit' to exit"
+                   userInput = gets.chomp
+                when "3"
+                   Apartment.all.each do |ap|
+                     puts "The following people live in apartment number #{ap.id}."
+                     Tenant.all.each do |t|
+                       if t.apartment_id == ap.id
+                         puts t.name
+                       end
+                     end
+                    # Tenant.all.each_index.select{|i| Tenant.all[i].apartment_id == ap.id}.each {|t| puts Tenant.all[t].name}
+                     puts "------------------------"
+                   end
+                   puts "You are back to the top menu, 'menu' for options, 'exit' to exit"
+                   userInput = gets.chomp
```

Comment:
These two lines show up a couple of places, you could create a method that does this to DRY up your code.